service/test/unit/config/test_leap.py

```diff
@@ -126,4 +127,53 @@ def test_should_add_welcome_email_on_a_fresh_account(self, mock_create_leap_sess
 
         self._user_bootstrap.setup(self.user_auth, self.password, '')
 
+        # TODO: this is incorrect
```

Comment:
If this is a message for the next person to pass through here, can you tell us why? If this is a message for future self, than was the future self supposed to be in the scope of this PR? :P